utils/build-presets.ini

```diff
@@ -169,6 +169,17 @@ skip-test-ios-host
 skip-test-tvos-host
 skip-test-watchos-host
 
+
+[preset: buildbot,tools=RA,stdlib=RDA,long_test]
+mixin-preset=
+    mixin_buildbot_tools_RA_stdlib_RDA
+
+test=0
+validation-test=0
+long-test=1
```

Comment:
I don't think you need `=1` here, so this should just be `long-test`. It may not seem like a big deal, but migrating `build-script-impl` arguments like this to Python ends up being tricky--Python's `argparse` does not allow numerical values for boolean command-line arguments. For more details, see https://github.com/apple/swift/pull/1745.